lambda/update/certificate_provider_sign.go

```diff
@@ -22,6 +23,9 @@ func (c CertificateProviderSign) Apply(lpa *shared.Lpa) []shared.FieldError {
 	lpa.CertificateProvider.Address = c.Address
 	lpa.CertificateProvider.SignedAt = &c.SignedAt
 	lpa.CertificateProvider.ContactLanguagePreference = c.ContactLanguagePreference
+	lpa.CertificateProvider.Email = c.Email
+	// to account for paper to online moves
+	lpa.CertificateProvider.Channel = shared.ChannelOnline
```

Comment:
should this be another update type, as in the story it says "When They use their code from the pack to move online"?

Reply:
I'd originally had it as a change but hard coded rather than coming over from MRLPA as it could only ever be online but yeah you're right we'd want to see that change from paper to online - I'll add the change back in 👍